inst/test-sites.csv

```diff
@@ -0,0 +1,42 @@
+site_id,site_name
```

Comment:
Site list was the output from [this query in the old permutation test script](https://github.com/PecanProject/pecan/blob/develop/base/workflow/inst/permutation_tests.R#L77-L104).

Agree that the prescribed table is a more efficient set of tests. My last commit here adds the capability to do that (not much to it -- the API makes all of this really easy).

Inputs are passed into `submit.workflow` as a list that matches the format of the XML file. Currently, the only input we vary here is met, which makes it easy to store in CSV format. There may be something clever we can do with regular expressions to get more stuff into the CSV...or, more straightforwardly, we could just add more columns for additional inputs. 

That reminds me -- all of the ED tests will fail because I'm not providing the other ED inputs. I'll add a commit to set the ones we never change as defaults.

Reply:
Thanks for the additions -- I think having two examples is even more helpful to folks. On our next call we should talk about filling in years, etc in that csv so that we don't rely just on the 2004 default met and other things that reflect inadequacies in the underlying data more than inadequacies in our system.

In terms of other inputs, I think adding more columns is the way to go. We can add many of them as tags (e.g. naming the source in the <soils> tag and checking that the soil processing workflow works for different sources) and set others to the appropriate input_id. Is the existing code smart enough to distinguish a source from an ID or do we need to add that to the API functions? We can add that to next week's agenda, but models and met are the clear priority.

Reply:
> Is the existing code smart enough to distinguish a source from an ID or do we need to add that to the API functions?

The submitting function just passes the literal list into the `<run> <inputs>` field ([source](https://github.com/PecanProject/rpecanapi/blob/master/R/submit.workflow.R#L49)), so we have full control. My current script just treats the "met" column as referring to source, but that's easy enough to modify.

Reply:
First pass at input processing script added in https://github.com/PecanProject/rpecanapi/pull/7/commits/092381228338ad36b83b21052992fbe96f4d2abf (also configures ED2 default inputs). Goal is to expand this later as we add more inputs.